transcode/manifest.go

```diff
@@ -176,5 +176,15 @@ func manifestURLToSegmentURL(manifestURL, segmentFilename string) (string, error
 		return "", fmt.Errorf("failed to parse segment filename when converting to segment URL: %s", err)
 	}
 
+	// If a manifest is using URL paths instead of relative paths, then override the
```

Comment:
This is not clear to me, why do we change absolute => relative paths only for Google GCP? What about other S3 storages, AWS, Storj?